### PR TITLE
[LAP-19] more physics integration

### PR DESCRIPTION
**Pull request by Algo-ryth-mix**

### CODE REVIEW

File: docs/source/physics-wiki/physics-index.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+collisions and other interesting physics related problems for your game.
+
+To use Diviner make sure to link against ``legion-physics.lib``
+If you are on Windows you can do this by right-clicking your project -> Properties -> Linker -> Input 
+`More Details <https://stackoverflow.com/a/4446085/5487005>`_
+
+Adding Physics Components
+-------------------------
+
+
```
Comment (Algo-ryth-mix) [inline docs/source/physics-wiki/physics-index.rst:43]: Need some input from @Developer-The-Great here

Comment (Developer-The-Great) [reply docs/source/physics-wiki/physics-index.rst:43]: looks pretty good, I'll add the adding physics component part later

**Review by Developer-The-Great — APPROVED** [review]

**Comment by Algo-ryth-mix** [comment]
@Developer-The-Great @GlynLeine this is ready for review now, please check it out tomorrow

**Comment by GlynLeine** [comment]
> @Developer-The-Great @GlynLeine this is ready for review now, please check it out tomorrow

will do

**Review by GlynLeine — APPROVED** [review]

**Pull request merged**